### ISSUE TITLE
Add `KeepAngle(var)` to Built-in Commands

### ISSUE THREAD

**Issue by MatiDragon-YT** [sannybuilder/dev#207]
This function should be in charge of maintaining the value of a floating variable, within 360 degrees, in the form of endless tape.
That is to say, that if it is passed of the 360, this one must pass to value 0 and vice versa.
```js
KeepAngle(@18) // float
/** Parameter:
 * 1 - var or lvar
*/
```
translate to
```lua
if @18 > 360.0
then @18 -= 360.0
end
if @18 < 0.0
then @18 += 360.0
end
```

**Comment by NicusorN5** [comment]
As far as I'm concerned, doesn't the GTA 3D games use radians?

**Comment by MatiDragon-YT** [comment]
@NicusorN5 Not natively, not with some opcode. But on the part of SAMPFuncs yes, 2 are added
```r
0C01: samp math 1@ = radian_to_angle 0@ // (float)
0C02: samp math $1 = angle_to_radian $1 // (float)
```

**Comment by x87** [comment]
can be implemented with #206
